src/sentry/models/integration.py

```diff
@@ -13,10 +13,26 @@
 from sentry.signals import integration_added
 
 
+class PagerDutyService(Model):
+    __core__ = False
+
+    organization_integration = FlexibleForeignKey("sentry.OrganizationIntegration")
+    integration_key = models.CharField(max_length=255)
+    service_id = models.CharField(max_length=255)
+    service_name = models.CharField(max_length=255)
```

Comment:
Are names capped to 255 in pagerduty?

Reply:
testing them out they are capped to 100 characters